UEBPrep/app/views/cards/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= social_share_button_tag("A CARD") %>
```

Comment:
Oh i see, it generates the shareable url, not post to the user's social account?